contracts/MetadataViews.cdc

```diff
@@ -298,4 +297,33 @@ pub contract MetadataViews {
             self.createEmptyCollection=createEmptyCollectionFunction
         }
     }
+
+    // A view to expose the information needed to showcase this NFT's collection
+    //
+    // This can be used by applications to give an overview and graphics of the NFT collection
+    // this NFT belongs to.
+    pub struct NFTCollectionDisplay {
```

Comment:
Should we add a "social links" field here as well? So then a collection can advertise its discord, twitter, etc from a central place? If so, I'd just make it a `{String: String}` and let folks specify